keyboards/rotary_numpad/info.json

```
@@ -0,0 +1,64 @@
+{
+    "manufacturer": "rabijl",
+    "keyboard_name": "rotary_numpad",
+    "maintainer": "rabijl",
+    "bootloader": "caterina",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "encoder": true
+    },
+    "matrix_pins": {
+        "cols": ["F5", "F4", "B5", "B4"],
+        "rows": ["F6", "F7", "B1", "B3", "B2", "B6"]
+    },
+    "processor": "atmega32u4",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_numpad_6x4": {
```

This layout do not match:
1. Community layout of same name
2. Pictured product in `readme.md`
```suggestion
        "LAYOUT_numpad_6x4": {
```